### PR TITLE
DEV-81 - sharing process in action bar is more clear

### PR DESCRIPTION
**Pull request by Byuljapcho**
**Description**
Some users were confused about how to add reviewers and they would try to add reviewers by clicking on the share button. But the share button only copies the shareable link to the plan to the user's clipboard. The button with a person icon is where users could add or remove reviewers. To eliminate this confusion in navigating the UI, the share button is moved to where button with person icon was and now contains two options: add reviewers OR get shareable link to the plan. When the user hovers on the [Copy Plan Link] button or Add Reviewers title, they can get brief description of each feature. 

**Implementation**
In Actionbar.tsx, I deleted the existing share button and replaced the button with person icon with share button icon. Then I added [Copy Plan Link] button to the pop up that appears when new share button is clicked. I added a hovering message with ToolTip from MUI library to the [Copy Plan Link] button. Then I went to Reviewers.tsx to add hovering message with MUI ToolTip to the Add Reviewers title. 

**Testing**
Tested locally. 

**Screenshots**
Screen Shot 2022-12-06 at 6 33 58 PM
Screen Shot 2022-12-06 at 6 34 14 PM
Screen Shot 2022-12-06 at 6 34 24 PM

### CODE REVIEW

**Review by josephined21 — APPROVED** [review]
Looks good!

File: lib/components/dashboard/Actionbar.tsx [hunk 1]
```diff
@@ -383,6 +371,20 @@ const Actionbar: FC<{ mode: ReviewMode }> = ({ mode }) => {
...
+                placement="left"
+                arrow
+              >
+                <Button
+                  onClick={onShareClick}
+                  variant="outlined"
+                  sx={{ width: '10rem', height: '2rem', mx: 1, my: 1 }}
+                  color="info"
+                >
+                  <div className="ml-1">Copy Plan Link</div>
```
Comment (mattliu-mygit) [inline lib/components/dashboard/Actionbar.tsx:385]: Let's make the wording more clear and the UI a bit better. Maybe look at some other sites for inspiration maybe. 

Maybe reword to "Share Plan Snapshot", as it creates a copy of a snapshot of the current plan to someone else's account rather than adding them as a reviewer (the other function), where they can see the most updated version of the plan on their reviewer dashboard screen.

Comment (mattliu-mygit) [reply lib/components/dashboard/Actionbar.tsx:385]: Also the linter is failing please check the github action on why the linter is failing.

**Review by mattliu-mygit — CHANGES_REQUESTED** [review]

**Comment by Byuljapcho** [comment]
Made the wording more clear and UI better according to @mattliu-mygit comments

Add reviewer on hover message
Share plan snapshot on hover message

**Comment by Byuljapcho** [comment]
Not sure why prettier is failing - it's the same weird problem with the two files I never edited as it was with DEV-39 
Screen Shot 2023-02-19 at 1 05 11 PM

**Comment by mattliu-mygit** [comment]
Can you add a hoverable tooltip (much like other parts of the app that give a description on what this share snapshot button does? This is because I foresee people not understanding what this button does from vocabulary, or misunderstanding because of the place where it's at.

**Comment by Byuljapcho** [comment]
> Can you add a hoverable tooltip (much like other parts of the app that give a description) on what this share snapshot button does? This is because I foresee people not understanding what this button does from vocabulary, or misunderstanding because of the place where it's at.

I already added a hoverable, would this work? 
Screen Shot 2023-03-06 at 12.00.53 PM

**Comment by mattliu-mygit** [comment]
Remove the parenthesis in the tooltip and you're good to merge! ^.^

**Review by mattliu-mygit — APPROVED** [review]

**Review by josephined21 — CHANGES_REQUESTED** [review]
hi miseok! sorry one small nitpick, could you just change the wording (remove the "it" in "make it their own") so that it is ". . . that they can edit and make their own"? everything else looks good, thank you so much~!

**Review by josephined21 — APPROVED** [review]
looks great! tysm miseok uwu

**Review by aryamanshodhan — APPROVED** [review]
Looks great to me!
